src/content/en/2020/resource-hints.md

```diff
@@ -362,24 +370,25 @@ So far only 0.77% websites adopted this new hint as Chrome is still [actively](h
 The largest use is with script elements, which is unsurprising as the number of JS primary and third-party files continues to grow.
 
 {{ figure_markup(
-  caption="Priority hints on mobile have \"low\" importance",
+  caption="The rate of priority hints on mobile that have \"low\" importance.",
   content="16.11%",
   classes="big-number",
   sheets_gid="1098063134",
   sql_file="priority_hints_by_element_and_importance.sql"
 ) }}
 
+{# TODO(editors): Could the use of defense/offence be unclear here? #}
```

Comment:
@rviscomi

Reply:
Agree, I'm not clear what "offense" means in this context.

Reply:
I tried to reword this section to be more clear, I'll highlight the new TODO with a comment.